tests/test-cases/include-local-wildcard/.gitlab-ci.yml

```diff
@@ -1,3 +1,3 @@
 ---
 include:
-  local: '*.gitlab-ci.yml'
```

Comment:
Currently this `include:local` will result in a recursive  inclusion since the `excludedGlob` have been removed